dist/testTrack.js

```diff
@@ -161,7 +161,8 @@
     })();
     
     var TestTrackConfig = (function() { // jshint ignore:line
-        var config,
+        var DEFAULT_TT_VISITOR_ID_COOKIE_NAME = 'tt_visitor_id',
```

Comment:
Should we name this the same as the environment variable in the rails client? `TEST_TRACK_VISITOR_COOKIE_NAME`?